packages/dd-trace/test/exporters/agent/request.spec.js

```diff
@@ -50,10 +50,11 @@ describe('request', () => {
       data: Buffer.from(JSON.stringify({ foo: 'bar' }))
     }, (err, res) => {
       expect(res).to.equal('OK')
+      done(err)
```

Comment:
I _think_ these were not being done correctly. I think if you want to make the test async by returning a value it needs to be a promise, which this one isn't. Using `done` is what the other tests in this file does